app/controllers/application_controller.rb

```diff
@@ -114,6 +115,18 @@ def invalid_auth_token
     redirect_to root_url
   end
 
+  def invalid_phone_number(exception)
+    flash[:error] = case exception.code.to_s
+                    when  Figaro.env.twilio_sms_error
```

Comment:
let's pull these into constants on `TwilioService` and not make them envs --- they're not going to change per-environment or anything